### DISCUSSION TITLE
Another alternative icon!

### DISCUSSION THREAD

**Discussion by diegobit** [kovidgoyal/kitty#7748] [Show and tell]
Hi all! I'll ride the wave of alternative icons for kitty with another one inspired by the whiskers idea, but with the colors of the original icon! Hope someone will like it!

[https://github.com/diegobit/kitty-icon](https://github.com/diegobit/kitty-icon)

![icon_128x128@2x](https://github.com/user-attachments/assets/a37d7830-4a8c-45a8-988a-3e98a41ea541)

**Comment by kovidgoyal** [comment]
Added to the FAQ:
https://github.com/kovidgoyal/kitty/commit/6d06415ec5531eafbd52488734afd6e28bed1759
